src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -37,7 +41,12 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator, IRBlobSe
         private volatile Task _runTask;
         private volatile Task<REvaluationResult> _cancelEvaluationAfterRunTask;
         private volatile Task<ulong> _cancelCreateBlobAfterRunTask;
-        private volatile Task<byte[]> _cancelGetBlobAfterRunTask;
+        private volatile Task<byte[]> _cancelBlobReadAllAfterRunTask;
+        private volatile Task<byte[]> _cancelBlobReadAfterRunTask;
```

Comment:
ulong is the blob id. the other methods return size of the content. .Net uses long to represent sizes in streams.